Automattic/sensei
# Fix: Redirect to message after logging in from message screen.

### Changes proposed in this Pull Request

Currently when you try to view a message while not logged in, you will get redirected to the `My Courses` page, which will include a login form.  Once you log in, it will take you to `/wp-admin`.  This might be different depending on the roles but at least that's what happend for an `Administrator` user.

If you're trying to view a message, I feel like it should redirect you back to that message once you're logged in.  So this adds that original message URL to the `redirect_to` path, and then redirects you there once you're logged in.  

### Testing instructions

* Apply this patch
* Log out or use an incognito mode
* Try to navigate to a message like URL like this sample one `http://sensei.local/messages/test-contact-teacher/`
* Notice you get a front-end login page, then once you log in you should be redirected to the actual message you were trying to see.

**Question**:  I changed the redirect to use `wp_safe_redirect` here, should we also change the call a few lines below to use `wp_safe_redirect`?
